webpack.config.babel.js

```diff
@@ -54,6 +54,16 @@ module.exports = ({ stats = false } = {}) => ({
         test: /\.css$/,
         use: ['style-loader', 'css-loader'],
       },
+      {
+        test: /\.(ttf|otf|eot|woff2?)(\?[a-z0-9]+)?$/,
+        use: {
+          loader: 'file-loader',
+          options: {
+            name: '[name].[ext]',
+            mimetype: 'application/font-otf',
```

Comment:
`mimetype` is for `url-loader`, file-loader doesn't care. Not to mention different fonts which all can be matched by the test regexp need different mimetypes. Also mimetype in url-loader is determined from file extension by default